### PR TITLE
chore: add Daytona plugin to microsite

### PR DESCRIPTION
**Pull request by adityasinghal26**
Signed-off-by: Adi-Sin <adiaggarwal26@gmail.com>

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks for the PR!

It looks like there's something wrong with the included SVG, probably the semicolon :)
Does your license for the image allow for it to be included in this repository?

**Review by jhaals — APPROVED** [review]

File: microsite/data/plugins/daytona.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+iconUrl: /img/daytona.svg
+npmPackageName: '@daytonaio/backstage-plugin-daytona'
+tags:
+  - developer-tools
+  - development-environment
+  - dev-env
+  - dev-environment-setup
+  - development-environment-manager
+  - cloud-development-environment
+  - CDE
```
Comment (jhaals) [inline microsite/data/plugins/daytona.yaml:17]:
```suggestion
  - Development
  - Tools
  - Productivity
```

I realized these are actually what populates the dropdown on the plugin page so let's try to stick to the already existing ones. I added a few that I think should fit :)

**Review by jhaals — CHANGES_REQUESTED** [review]

File: microsite/data/plugins/daytona.yaml (hunk 1 above)
Comment (adityasinghal26) [reply microsite/data/plugins/daytona.yaml:17]: Thanks for the suggestion. This seems to fit well. I'll get it double checked internally and update the PR based on your suggestion.

Comment (adityasinghal26) [reply microsite/data/plugins/daytona.yaml]: @jhaals the suggestions are taken care of and committed. Can you check it and approve the same to merge?

**Review by jhaals — APPROVED** [review]

**Pull request merged**